.github/dependabot.yml

```diff
@@ -44,7 +44,6 @@ updates:
   open-pull-requests-limit: 10
 - package-ecosystem: gradle
```

Comment:
I believe we now have duplicate entries and the v3 entries should entirely be removed